### PR TITLE
Your First Service Guides - Adaptation for V2 service definition

### PR DESCRIPTION
**Pull request by marians**
Update for all "Your first App" guides

See also
- https://github.com/giantswarm/giantswarm-firstapp-java/pull/12
- https://github.com/giantswarm/giantswarm-firstapp-nodejs/pull/19
- https://github.com/giantswarm/giantswarm-firstapp-php/pull/10
- https://github.com/giantswarm/giantswarm-firstapp-python/pull/7
- https://github.com/giantswarm/giantswarm-firstapp-go/pull/14

### CODE REVIEW

**Comment by marians** [comment]
RFR @zyndiecate @webwurst

File: content/guides/your-first-service/golang.md [hunk 1]
```diff
@@ -184,70 +183,63 @@ Still assuming that your username is `yourusername`, you can now push the image
 $ docker push registry.giantswarm.io/yourusername/currentweather-go
 ```
 
-### Configuring your application
+### Defining your service
 
-Applications in Giant Swarm are [configured using a JSON configuration file](/reference/swarm-json/) that is usually called `swarm.json`. For this application, we create a configuration containing one service with two components.
+Services in Giant Swarm are [defined using a JSON file](/reference/swarm-json/) that is usually called `swarm.json`. For this service, we two components.
```
Comment (zyndiecate) [inline content/guides/your-first-service/golang.md]: `we two components` -> `with two components`

File: content/guides/your-first-service/nodejs.md [hunk 2]
```diff
@@ -173,92 +173,85 @@ Still assuming that your username is `yourusername`, you can now push the image
 $ docker push registry.giantswarm.io/yourusername/currentweather-nodejs
 ```
 
-### Configuring your application
+### Defining your service
 
-Applications in Giant Swarm are [configured using a JSON configuration file](/reference/swarm-json/) that is usually called `swarm.json`. For this application, we create a configuration containing one service with two components.
+Services in Giant Swarm are [defined using a JSON file](/reference/swarm-json/) that is usually called `swarm.json`. For this service, we two components.
```
Comment (zyndiecate) [inline content/guides/your-first-service/nodejs.md]: `we two components` -> `with two components`

File: content/guides/your-first-service/php.md [hunk 3]
```diff
@@ -192,70 +192,63 @@ Still assuming that your username is `yourusername`, you can now push the image
 $ docker push registry.giantswarm.io/yourusername/currentweather-php
 ```
 
-### Configuring your application
+### Defining your service
 
-Applications in Giant Swarm are [configured using a JSON configuration file](/reference/swarm-json/) that is usually called `swarm.json`. For this application, we create a configuration containing one service with two components.
+Services in Giant Swarm are [defined using a JSON file](/reference/swarm-json/) that is usually called `swarm.json`. For this service, we two components.
```
Comment (zyndiecate) [inline content/guides/your-first-service/php.md]: `with two components`

File: content/guides/your-first-service/python.md [hunk 4]
```diff
@@ -179,70 +179,63 @@ Still assuming that your username is `yourusername`, you can now push the image
 $ docker push registry.giantswarm.io/yourusername/currentweather-python
 ```
 
-### Configuring your application
+### Defining your service
 
-Applications in Giant Swarm are [configured using a JSON configuration file](/reference/swarm-json/) that is usually called `swarm.json`. For this application, we create a configuration containing one service with two components.
+Services in Giant Swarm are [defined using a JSON file](/reference/swarm-json/) that is usually called `swarm.json`. For this service, we two components.
```
Comment (zyndiecate) [inline content/guides/your-first-service/python.md]: with to components :facepunch:

File: content/guides/your-first-service/ruby.md [hunk 5]
```diff
@@ -176,92 +176,85 @@ Still assuming that your username is `yourusername`, you can now push the image
 $ docker push registry.giantswarm.io/yourusername/currentweather-ruby
 ```
 
-### Configuring your application
+### Defining your service
 
-Applications in Giant Swarm are [configured using a JSON configuration file](/reference/swarm-json/) that is usually called `swarm.json`. For this application, we create a configuration containing one service with two components.
+Services in Giant Swarm are [defined using a JSON file](/reference/swarm-json/) that is usually called `swarm.json`. For this service, we two components.
```
Comment (zyndiecate) [inline content/guides/your-first-service/ruby.md]: ...

**Comment by zyndiecate** [comment]
Minors. LGTM.

File: content/guides/your-first-service/golang.md (hunk 1 above)
Comment (marians) [inline content/guides/your-first-service/golang.md]: Resolved

File: content/guides/your-first-service/nodejs.md (hunk 2 above)
Comment (marians) [inline content/guides/your-first-service/nodejs.md]: Resolved

File: content/guides/your-first-service/php.md (hunk 3 above)
Comment (marians) [inline content/guides/your-first-service/php.md]: Resolved

**Comment by marians** [comment]
Thanks Tim! I'll ask around for a second check, since so many things have changed that it's easy to miss something.

File: content/guides/your-first-service/golang.md [hunk 6]
```diff
@@ -59,11 +59,11 @@ We won't repeat the `sudo` note for the sake of readability of the rest of this
...
-## Overview of our application
+## Overview of our service
 
-This diagram depicts how our application components will be set up.
+This diagram depicts how our service components will be set up.
 
-![Application schema diagram](/img/your-first-application-schema-go.png)
+![Application schema diagram](/img/your-first-service-schema-go.png)
 
 We have one component which we call `webserver` as the core piece. It will provide a Go HTTP server. When accessed by a user, it should display the current weather at our home town, Cologne/Germany.
```
Comment (hectorj2f) [inline content/guides/your-first-service/golang.md]: I am not sure about the excessive use of `component` term which can be a legacy term in some cases. Perhaps we should use the term `node` which wasn't mentioned in here. What is your opinion on that ?

File: content/guides/your-first-service/ruby.md [hunk 7]
```diff
@@ -176,92 +176,85 @@ Still assuming that your username is `yourusername`, you can now push the image
...
 So here is what we get when doing so (your output will vary slightly):
 
 ```nohighlight
-App currentweather-app is up
+Service currentweather is up
 
-service                 component           instanceid    created              status
-currentweather-service  ruby-web-component  TdfsdckAwoS3  2015-02-10 07:20:03  up
-currentweather-service  redis               VrvpFsjGOLyq  2015-02-10 07:20:02  up
+component  instanceid    created              status
```
Comment (hectorj2f) [inline content/guides/your-first-service/ruby.md]: We miss the image column in all the examples so I guess it's on purpose, just to confirm! :+1:

**Comment by hectorj2f** [comment]
Minor comment to be discussed, after that LGTM

**Comment by marians** [comment]
Updated for the node -> component change

**Comment by hectorj2f** [comment]
LGTM

**Comment by marians** [comment]
Also added the image column to swarm status output and fixed one image name in the definition

**Comment by hectorj2f** [comment]
:+1:

**Pull request merged**